guide/07-working-with-bigdata/09-managing_big_data.ipynb

```diff
@@ -0,0 +1,1644 @@
+{
```

Comment:
<p>The <code>arcgis.geoanalytics.manage_data</code> submodule contains functions that are used for the day-to-day management of geographic and tabular data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>